hack/local-up-cluster.sh

```diff
@@ -986,6 +995,10 @@ Logs:
 EOF
 fi
 
+if [[ "${ENABLE_APISERVER_DYNAMIC_AUDIT:-}" = true ]]; then
+  echo " running with dynamic audit"
```

Comment:
Did you intend to leave this here? If so, would it make more sense to put this up in the block starting at line 521?